packages/client/src/Feed.js

```diff
@@ -46,7 +46,7 @@ const Feed = () => {
           />
         );
       })}
-      <Footer isAdmin={isAdmin} />
+      <Footer isAdmin={isAdmin} setEditSubmitted={setEditSubmitted} logout={logout}/>
```

Comment:
Why not just have footer `useContext(AuthContext)`

Reply:
Gotcha!